### ISSUE TITLE
Run bert on Windows+ ARC750, extremely long first interence  time (over 300s) 

### ISSUE THREAD

**Issue by juan-OY** [intel-analytics/BigDL#9187]
Setup the windows system to run Bert on ARC
Device:  i7-12700 (with iGPU) + dGPU ARC750
Win10 LTS
Driver version: Latest 31.0.101.4887

Steps to reproduce:
安装arc驱动
安装最新的visual studio（oneapi要求要先装visual studio）
安装oneapi，，安装的时候全部默认选项就行
conda create -n <env_name> python=3.9 libuv
安装图里的torch和ipex的包
安装 bigdl-llm==2.4.0b20231009
安装图里的bigdl_core_xe那个包

1st inference costs long time as 309 seconds.
 Execution log: 
(arcllm) C:\work\bert>python generate_optimze_model_arc.py
1st time bigdl framework inference time in (s) 309.55492997169495
Iteration 1: native framework inference time in (s) 0.04679560661315918
Iteration 2: native framework inference time in (s) 0.015627622604370117
Iteration 3: native framework inference time in (s) 0.031246662139892578
Iteration 4: native framework inference time in (s) 0.03124690055847168
Iteration 5: native framework inference time in (s) 0.03124237060546875
Iteration 6: native framework inference time in (s) 0.015628337860107422
Iteration 7: native framework inference time in (s) 0.03124690055847168
Iteration 8: native framework inference time in (s) 0.031242847442626953
Iteration 9: native framework inference time in (s) 0.01562786102294922
Iteration 10: native framework inference time in (s) 0.03124713897705078
Average inference time over 10 iterations: 0.028115224838256837 seconds

---------------------------------------------
Code to reproduce:
---------------------------------------------

import intel_extension_for_pytorch as ipex
import time
from transformers import BertTokenizer, BertModel
from bigdl.llm import optimize_model

tokenizer = BertTokenizer.from_pretrained('bert_large_uncased')
model = BertModel.from_pretrained("bert_large_uncased")

# With only one line to enable BigDL-LLM optimization on model
model = optimize_model(model)
model = model.to('xpu:0')
text = "I want to get a hello word c++ sample."

# Define the number of iterations
num_iterations = 10
total_time = 0

encoded_input = tokenizer(text, return_tensors='pt').to('xpu:0')
t0= time.time()
output = model(**encoded_input)
print("1st time bigdl framework inference time in (s)", time.time()-t0)
#print("output: ", output)
for i in range(num_iterations):
    #encoded_input = tokenizer(text, return_tensors='pt')
    t0 = time.time()
    output = model(**encoded_input)
    inference_time = time.time() - t0
    total_time += inference_time
    print(f"Iteration {i + 1}: native framework inference time in (s) {inference_time}")

average_time = total_time / num_iterations
print(f"Average inference time over {num_iterations} iterations: {average_time} seconds")

**Comment by MeouSker77** [comment]
yes, the long first inference time is as expected, it seems ipex will generate real machine instructions during the first inference on Windows (just like JIT), a future version may fix this problem
